modules/virt-configuring-configmap-for-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-configuring-configmap-for-obsolete-cpu-models_{context}"]
+= Configuring a ConfigMap for obsolete CPU models
+
+Use this procedure to configure a ConfigMap for obsolete CPU models.
+
+.Procedure
+
+* Create a ConfigMap object, specifying the obsolete CPU models in the `obsoleteCPUs` array. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <cpu-plugin-configmap> <1>
```

Comment:
@ksimon1 @omeryahud is the SSP operator creating this configmap by default?

Reply:
Yes

Reply:
Change was incorporated.